libs/coin-framework/src/account/helpers.test.ts

```diff
@@ -38,40 +33,13 @@ describe(getAccountCurrency.name, () => {
     });
   });
 
-  describe("given a ChildAccount", () => {
-    it("should return the currency", () => {
-      const sampleCurrency = { family: "bitcoin" } as CryptoCurrency;
-      childAccount.currency = sampleCurrency;
-      expect(getAccountCurrency(childAccount)).toEqual(sampleCurrency);
-    });
-  });
-
   describe("given a TokenAccount", () => {
     it("should return the token currency", () => {
       const sampleToken = { id: "tokenId" } as TokenCurrency;
       tokenAccount.token = sampleToken;
       expect(getAccountCurrency(tokenAccount)).toEqual(sampleToken);
     });
   });
-
-  describe("given an unknown type Account", () => {
```

Comment:
I don't understand why you remove those tests. They are limited to ChildAccount (it is not obvious)?

Reply:
(answered above)